routes/w3c/group.js

```diff
@@ -119,18 +135,37 @@ async function getPatentPolicy(activeCharterApiUrl) {
   }
 }
 
-/** @param {string} shortname */
-function getGroupMeta(shortname) {
-  /** @type {number} */
-  let id;
-  /** @type {Group["type"]} */
-  let type;
-
-  if (groups.wg.hasOwnProperty(shortname)) {
-    [type, id] = ["wg", groups.wg[shortname]];
-  } else if (groups.cg.hasOwnProperty(shortname)) {
-    [type, id] = ["cg", groups.cg[shortname]];
+/**
+ * @param {string} shortname
+ * @param {Group["type"]} [requestedType]
+ * */
+function getGroupMeta(shortname, requestedType) {
+  /** @type {Group["type"][]} */
+  const types = requestedType ? [requestedType] : ["wg", "cg"];
+  const data = types
+    .map(type => {
+      if (groups[type].hasOwnProperty(shortname)) {
+        /** @type {number} */
+        const id = groups[type][shortname];
+        return { shortname, type, id };
+      }
+    })
+    .filter(g => g);
+
+  switch (data.length) {
+    case 1:
+      return data[0];
+    case 0: {
+      const msg = `No group with shortname: "${shortname}"${
+        requestedType ? ` and type: "${requestedType}"` : ""
+      }`;
+      throw new HTTPError(404, msg);
+    }
+    default: {
+      const suggestions = data.map(g => `"${shortname}/${g.type}"`).join(", ");
+      const hint = `Please specify one of following: ${suggestions}`;
+      const msg = `Multiple groups with shortname: "${shortname}".`;
+      throw new HTTPError(409, `${msg} ${hint}`);
```

Comment:
"409 Conflict" might not be semantically correct here :thinking: 
```suggestion
      throw new HTTPError(404, `${msg} ${hint}`);
```